sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -8,6 +8,14 @@
 
 ### Bugs Fixed
 
+* Fixed an issue where a `DuplicateKeyException` could be thrown if `EventId`
+  and `EventName` were present in both `LogRecord` (`LogRecord.EventId`,
+  `LogRecord.EventName`) and `LogRecord.Attributes`. The method now checks if
+  these keys exist before attempting to add them, ensuring that `EventId` and
```

Comment:
the priority order is not clear.. if attributes and LogRecord both have eventid/name, which one wins?